it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -56,94 +56,94 @@
 12505=Caricamento allegato in corso…
 
 ## @name NS_MSG_COULDNT_OPEN_FCC_FOLDER
-12506=Impossibile aprire la cartella Posta inviata. Verificare che le impostazioni di posta siano corrette.
+12506=Impossibile aprire la cartella Posta inviata. Verifica che le tue impostazioni di posta siano corrette.
 
 ## @name NS_MSG_CANT_POST_TO_MULTIPLE_NEWS_HOSTS
-12507=Non è possibile inviare verso più server di gruppi di discussione. Riprovare.
+12507=Non è possibile pubblicare su host di notizie multipli. Riprova.
 
 ## @name NS_MSG_ASSEMB_DONE_MSG
-12508=Ricomposizione messaggio…completato.
+12508=Unione messaggio in corso… Completata.
 
 ## @name NS_MSG_ASSEMBLING_MSG
-12509=Ricomposizione messaggio…
+12509=Unione messaggio in corso…
 
 ## @name NS_MSG_NO_SENDER
-12510=Non è stato specificato il mittente. Inserire il proprio indirizzo e-mail nelle impostazioni di Posta e gruppi di discussione.
+12510=Non è stato specificato il mittente. Inserisci il tuo indirizzo e-mail nelle impostazioni account E-mail e Newsgroup.
 
 ## @name NS_MSG_NO_RECIPIENTS
-12511=Non sono stati specificati destinatari. Inserire un destinatario o un gruppo di discussione nella sezione degli indirizzi.
+12511=Non sono stati specificati destinatari. Inserisci un destinatario o un newsgroup nell'area indirizzo.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12511=Non sono stati specificati destinatari. Inserisci un destinatario o un newsgroup nel campo indirizzo.